gpt_index/indices/query/query_runner.py

```diff
@@ -74,9 +77,10 @@ def query(
         self, query_str_or_bundle: Union[str, QueryBundle], index_struct: IndexStruct
     ) -> Response:
         """Run query."""
+        # NOTE: query processor is only run once
+        # TODO: Consider refactor to support index-specific query processing
```

Comment:
was going to ask this, but this makes sense. i'm potentially interested in how we can do query processing given contents of the nodes themselves

Reply:
Yep, going to keep this PR small. Definitely interesting future work to improve query processing with node content (and or metadata).